### PR TITLE
Fix of division by zero error

### LINKED ISSUE #1: Division by zero crashes the program
I've put following input and it crashes the program

```
Kąkuter
Type your calculations [type !help to access help panel, or type quit, to close script]asdasd
Type your calculations [type !help to access help panel, or type quit, to close script]3+2
5/0
No No No
5+2/0
Traceback (most recent call last):
  File "main.py", line 164, in <module>
    kalkulacje()
  File "main.py", line 108, in kalkulacje
    kalkulacje()
  File "main.py", line 160, in kalkulacje
    myeval2(UserIn, previous)
  File "main.py", line 43, in myeval2
    return eval(str(p) + w)
  File "<string>", line 1, in <module>
ZeroDivisionError: division by zero
```

It looks that there's designed protection against that, but it works only if division by zero is the single operation

Please assign me to review pull request before merging

### PR DESCRIPTION
**Pull request by WhiteCrosses**
- added ZeroDivisionError fix

```
def zero_error():
    print("Dont divide by 0!")

def myeval1(s):
    try:
        return eval(s)
    except SyntaxError:
        return 0
    except ZeroDivisionError:
        zero_error()
        return 0

def myeval2(w, p):
    try:
        return eval(str(p) + w)
    except SyntaxError:
        return 0
    except ZeroDivisionError:
        zero_error()
        return 0

```

- Fixed minor bug of not returning to home after viewing certain paths in help panel

### CODE REVIEW

**Review by maziu — CHANGES_REQUESTED** [review]
Fix seems to be done right, however you unnecessary created new file. 
Correct workflow would be to make fixes **on the same file**, instead of creating new file - that's main idea behind the git ;) Then during review I can see exact lines with changes and you can merge the branches into the same file. 
Why it's so important? If you'll leave two files, after merging them into main branch there will be two files. And we have one program, not two in this project. Imagine what happens if there is big project and everyone collaborating on it create their own files with their changes.
 
To sum up - please make commit into this branch which:
- removes the file "-by-zero-fix"
- have changes in the original file

Then I can review and accept it

**Review by maziu — APPROVED** [review]
Fix OK, feel free to merge 👍
